dev.fredol.open-tv.yaml

```diff
@@ -0,0 +1,1152 @@
+id: dev.fredol.open-tv
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: open_tv
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --filesystem=host:ro
+  - --socket=pulseaudio
+  - --filesystem=/tmp:rw
```

Comment:
why is /tmp needed ?

Reply:
That's from mpv, I'm not really sure

Reply:
does it  work without it? Then you can drop it. I don't think MPV needs it.

Reply:
after testing and looking through mpv's source code, it's not needed. I removed it. Thanks